task1/flask_app_config.ini

```diff
@@ -0,0 +1,16 @@
+[flask]
+host = 127.0.0.1
+port = 5000
+secret_key =
+
+[general]
+logs_dir = logs/
+log_path_1 = logs/logger_1.log
```

Comment:
@patrikprovaznik try and use interpolation rather the repeating path -> https://docs.python.org/3/library/configparser.html#interpolation-of-values